images/Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM registry.access.redhat.com/ubi9/go-toolset:1.18 AS build
-WORKDIR $APP_ROOT/src
-COPY . .
-RUN make
+FROM registry.access.redhat.com/ubi9/go-toolset:1.22.9 AS build
```

Comment:
I would have expected that changing this line `FROM registry.access.redhat.com/ubi9/go-toolset:1.22.9 AS build` is enough to fix `make image`?
I'd use `latest` rather than a hardcoded version so that we don't need to update it too often.

Reply:
I was running into errors while running the script so I put the fixes.
Changed the tag to `latest`

Reply:
If I understand correctly, your changes allow to build from the `images` directory, eg `cd images; podman build .`
`make image` on the other hand attempts to build from the top level directory, not from `images`.
With your changes, this no longer works.

I'm fine with building from `images` as long as `make image` is updated and still works. It would be worth adding a check in github actions that the image can always be built, this does not seem to be tested at the moment.

Reply:
yes, you're right - thanks! I changed the Dockerfile.

> It would be worth adding a check in github actions that the image can always be built, this does not seem to be tested at the moment.

[Something like this? (adding a target in the existing build check)](https://github.com/containers/gvisor-tap-vsock/pull/460/commits/3dc6f02eca57296e11404c6bd017c3a59c56131b) or would it be better to have a separate check?

Reply:
Adding to the existing build check should be good.